.storybook/webpack.config.ts

```diff
@@ -31,20 +31,19 @@ switch (process.env.ENV) {
 }
 
 const env = dotenv.config({path: path.resolve(__dirname, `../${envFile}`)});
-const custom = require('../config/webpack/webpack.common').default({
+const custom: CustomWebpackConfig = require('../config/webpack/webpack.common')({
     envFile,
 });
 
-module.exports = ({config}) => {
-    config.resolve.alias = {
-        'react-native-config': 'react-web-config',
-        'react-native$': 'react-native-web',
-        '@react-native-community/netinfo': path.resolve(__dirname, '../__mocks__/@react-native-community/netinfo.ts'),
-        '@react-navigation/native': path.resolve(__dirname, '../__mocks__/@react-navigation/native'),
-
-        // Module alias support for storybook files, coping from `webpack.common.ts`
-        ...custom.resolve.alias,
-    };
+const webpackConfig = ({config}: {config: Configuration}) => {
+    if (config.resolve && config.plugins && config.module) {
```

Comment:
Even if any of the config prop is not set we should still set our props.

```suggestion
    if (!config.resolve) {
        config.resolve = {};
    }
    if (!config.plugins) {
        config.plugins = [];
    }
    if (!config.module) {
        config.module = {};
    }
```